include/cpuinfo.h

```diff
@@ -2094,6 +2121,40 @@ static inline bool cpuinfo_has_riscv_v(void) {
 #endif
 }
 
+#if CPUINFO_ARCH_PPC64
+	struct cpuinfo_powerpc_isa {
+		bool vsx;
+		bool htm;
+		bool mma;
+	};
+
+	extern struct cpuinfo_powerpc_isa cpuinfo_isa;
+#endif
+
+static inline bool cpuinfo_has_powerpc_vsx(void) {
+#if CPUINFO_ARCH_PPC64
+	return cpuinfo_isa.vsx;
+#else
+	return false;
+#endif
+}
+
+static inline bool cpuinfo_has_powerpc_htm(void) {
+#if CPUINFO_ARCH_PPC64
+	return cpuinfo_isa.htm;
+#else
+	return false;
+#endif
+}
+
+static inline bool cpuinfo_has_powerpc_mma(void) {
+#if CPUINFO_ARCH_PPC64
+	return cpuinfo_isa.mma;
+#else
+		return false;
```

Comment:
Indent is wrong in else cycle
```suggestion
	return false;
```

Reply:
I have removed extra space in new commit